api/worker/handlers.go

```diff
@@ -0,0 +1,99 @@
+package worker
+
+import (
+	"crypto/rsa"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/ufcg-lsd/arrebol-pb/api"
+	"github.com/ufcg-lsd/arrebol-pb/crypto"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+const SIGNATURE_HEADER string = "SIGNATURE";
+
+type WorkerSpec struct {
```

Comment:
Do we need to make sure the way the fields are written? I think in the worker it is Id, for example...

Reply:
Fixed.